File: JAVA/EJERCICIOS/BUCLES/Ejercicio28SumarNumeros.java
package JAVA.EJERCICIOS.BUCLES;
/*
Escribe un programa que pida al usuario números 
hasta que ingrese el número 0. Al final, debe
imprimir la suma de todos los números ingresados.
 */

import java.util.Scanner;
public class Ejercicio28SumarNumeros {
    static Scanner leer = new Scanner(System.in);
    public static void main(String[] args) {
        // variables
        int numero = 1, suma = 0;
        // mientras que el número sea distinto de 0
        while (numero != 0) {
            // pedimos el numero y incrementamos la suma con cada numero ingresado
            System.out.print("Ingrese un numero: ");
            numero = leer.nextInt();
            suma += numero;
        }
        // mostramos la suma total
        System.out.println("Suma total: " + suma);
    }
}

File: PYTHON/EJERCICIOS/BUCLES/Ejercicio28-sumar-numeros.py
"""
EJERCICIO DE LOGICA DE PROGRAMACIÓN:
Escribe un programa que pida al usuario números 
hasta que ingrese el número 0. Al final, debe
imprimir la suma de todos los números ingresados.
"""
# variables 
numero,suma = 1,0
# mientras que el NUMERO sea != de 0 pedimos numeros 
# incrementamos la variable suma
while numero != 0:
    numero = int(input("Ingresa un número: "))
    suma+=numero
# mostramos la suma 
print(f"Suma total: {suma}")